src/canvas-extractor/edfi_canvas_extractor/mapping/submissions.py

```diff
@@ -19,12 +19,14 @@ def _get_date_formated(date) -> str:
 
 
 def _get_status(row: pd.Series):
-    if row["late"] == 1:
+    if row["late"] == 'True':
         return "late"
-    if row["missing"] == 1:
+    if row["missing"] == 'True':
         return "missing"
-    if row["graded_at"] is not None:
+    if pd.isnull(row["graded_at"]) is False:
         return "graded"
```

Comment:
We no longer have "on-time". And that's ok - it looks like we had two different [descriptors](https://github.com/Ed-Fi-Alliance-OSS/LMS-Toolkit/blob/main/extension/Descriptors/SubmissionStatusDescriptor.xml) for the same thing: "graded" and "on-time". 

In Analytics Middle Tier, both status are correctly mapped to the "On time" descriptor constant, so that's good.

Can you go ahead and remove the "on-time" Descriptor from the XML file? Or remove "graded" and change above to "on-time"? I don't have a preference between the two.